src/robot/conf/settings.py

```diff
@@ -115,7 +115,12 @@ def _process_value(self, name, value):
             return utils.abspath(value)
         if name in ['SuiteStatLevel', 'MonitorWidth']:
             return self._convert_to_positive_integer_or_default(name, value)
-        if name in ['Listeners', 'VariableFiles']:
+        if name == 'Listeners':
+            if isinstance(value, list):
+                return [self._split_args_from_name_or_path(item) for item in value]
+            else:
+                return [(value, [])]
+        if name == 'VariableFiles':
             return [self._split_args_from_name_or_path(item) for item in value]
```

Comment:
Hmm, it would actually be useful to be able to pass also variable files as objects. Definitely a subject to a separate enhancement request, though.